is/simple/deployment.yaml

```diff
@@ -0,0 +1,366 @@
+            <SvnUser>username</SvnUser>
+            <SvnPassword>password</SvnPassword>
+            <SvnUrlAppendTenantId>true</SvnUrlAppendTenantId>
+        </DeploymentSynchronizer>
+
+        <ServerInitializers>
+
+        </ServerInitializers>
+
+        <RequireCarbonServlet>${require.carbon.servlet}</RequireCarbonServlet>
+
+        <StatisticsReporterDisabled>true</StatisticsReporterDisabled>
+
+        <FeatureRepository>
+          <RepositoryName>default repository</RepositoryName>
+          <RepositoryURL>http://product-dist.wso2.com/p2/carbon/releases/wilkes/</RepositoryURL>
+        </FeatureRepository>
+
+       <APIManagement>
+
+      <Enabled>true</Enabled>
+
+      <LoadAPIContextsInServerStartup>true</LoadAPIContextsInServerStartup>
+       </APIManagement>
+    </Server>
+kind: ConfigMap
+metadata:
+  name: identity-server-conf
+  namespace: wso2
+
+---
+# Deploying kubernetes services
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: wso2is-service
+  namespace: wso2
+spec:
+  selector:
+    deployment: wso2is
+  ports:
+    - name: servlet-http
+      port: 9763
+      targetPort: 9763
+      protocol: TCP
+    - name: servlet-https
+      port: 9443
+      targetPort: 9443
+      protocol: TCP
+
+#SLEEP 10S
+
+---
+
+# Deploying wso2 identity server
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2is-deployment
+  namespace: wso2
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      deployment: wso2is
+  minReadySeconds: 30
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        deployment: wso2is
+    spec:
+      containers:
+        - name: wso2is
+          image: docker.wso2.com/wso2is:5.7.0
+          livenessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - nc -z localhost 9443
+            initialDelaySeconds: 180
+            periodSeconds: 10
+          readinessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - nc -z localhost 9443
+            initialDelaySeconds: 180
+            periodSeconds: 10
+          lifecycle:
+            preStop:
+              exec:
+                command:  ['sh', '-c', '${WSO2_SERVER_HOME}/bin/wso2server.sh stop']
+          imagePullPolicy: Always
+          securityContext:
+            runAsUser: 802
+          ports:
+            - containerPort: 9763
+              protocol: TCP
+            - containerPort: 9443
+              protocol: TCP
+          volumeMounts:
+            - name: identity-server-conf
+              mountPath: /home/wso2carbon/wso2-config-volume/repository/conf
+
+      initContainers:
+        - name: init-wso2is
+          image: busybox
+          command: ['sh', '-c', 'until nslookup identity-server-conf-axis2; do echo waiting for configs; sleep2; done']
```

Comment:
Since it's only one container, do we need any dependency? Also the lookup should be for some valid endpoint.